app/Http/Controllers/TeamController.php

```diff
@@ -105,11 +112,19 @@ public function destroy(Team $team) {
 	 * @param array $attributes
 	 * @return \App\Course
 	 */
-	private function createTeam(array $attributes) {
-		return Team::create([
+	private function createTeam(array $attributes, $image) {
+		$team = Team::create([
 			'name' => $attributes['teamName'],
 			'img_url' => $attributes['teamImage'],
```

Comment:
No estoy seguro de que esto se traduzca al URL

Reply:
Leftovers de la versión anterior. This shall be purged.